src/extension.ts

```diff
@@ -1,23 +1,53 @@
 import * as vscode from 'vscode';
-import neo4j, { Driver } from 'neo4j-driver';
+import neo4j, { AuthToken, Driver } from 'neo4j-driver';
 
 let driver: Driver;
+let selectedDb: any;
 
-export function activate(context: vscode.ExtensionContext) {
-	driver = neo4j.driver('neo4j://localhost', neo4j.auth.basic('neo4j', 'pass'), {
-		useBigInt: true
-	});
+export async function activate(context: vscode.ExtensionContext) {
+	function getConfiguredDbs(): any[] {
+		const config = vscode.workspace.getConfiguration('neo4j-cypher-runner');
+		return config.get('databases') || [];
+	}
+
+	async function performDatabaseSelection() {
+		const databases = getConfiguredDbs() || [];
+		if (databases.length > 0) {
+			const pickedConfig = await vscode.window.showQuickPick(
+				databases.map(db => db.name),
+				{
+					canPickMany: false,
+					title: 'Select Neo4j database',
+				}
+			);
+			selectedDb = databases.find(db => db.name === pickedConfig);
+			configureDriver();
+		} else {
+			vscode.window.showWarningMessage('No database configured');
```

Comment:
warning or error?

Reply:
It could be an error, since it's a blocker.